src/app/drop-down/drop-down.sample.ts

```diff
@@ -123,9 +123,9 @@ export class DropDownSampleComponent implements OnInit {
         this.igxDropDown.toggle(overlaySettings);
     }
 
-    onSelection() {
+    onSelection(event) {
```

Comment:
Is this change intentional?

Reply:
Yes, the demos weren't building with --prod --aot=true. This is just in the demos.